config/application.php

```diff
@@ -181,18 +181,18 @@
 // The hostname of the Redis server.
 Config::define('WP_REDIS_HOST', env('WP_REDIS_HOST'));
 // Set the connection scheme.
-Config::define('WP_REDIS_SCHEME', env('WP_REDIS_SCHEME') ?: 'rediss');
-// The prefix used for all cache keys to avoid data collisions, should be human readable
-Config::define('WP_REDIS_PREFIX', env('WP_REDIS_PREFIX') ?: 'local');
+Config::define('WP_REDIS_SCHEME', env('WP_REDIS_SCHEME') ?: 'tls');
+// The prefix used for all cache keys to avoid data collisions, should be human readable.
+Config::define('WP_REDIS_PREFIX', env('WP_REDIS_PREFIX') ?: WP_ENV);
 // Relay is the fastest redis client.
-Config::define('WP_REDIS_CLIENT', env('WP_REDIS_CLIENT') ?: 'predis'); 
-
-// Config::define('WP_REDIS_PASSWORD', env('WP_REDIS_PASSWORD')); 
+Config::define('WP_REDIS_CLIENT', env('WP_REDIS_CLIENT') ?: 'relay');
```

Comment:
The answer to this and your question about `WP_REDIS_PREFIX ` is ... it depends 😁

During stress testing we could trial out the 3 options `predis|phpredis|relay`.

On dev we can exec into the pod and make live changes to work out which config. is optimal.

On staging, because we have multiple pods, the quickest way to change the setting would be:

```
kubectl edit configmap intranet-staging -n $NSP
kubectl rollout restart deployment/intranet-staging -n $NSP
```

I agree, that after stress testing we should remove these :)